pubspec.yaml

```diff
@@ -11,7 +11,8 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_svg: '0.23.0+1'
-  provider: ^6.0.1
+  flutter_switch: 0.3.2
+  provider: 6.0.0
```

Comment:
We need to use at least version 6.0.1 of provider (same in `example/pubspec.yaml`) because this is what's being used in `xayn_architecture`. If we don't do this, there will be conflicts in other projects that import both `xayn_design` and `xayn_architecture`.

Reply:
fixed